packages/table/preview/index.tsx

```diff
@@ -644,6 +646,13 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
         this.maybeLogCallback(`[onCopy] success = ${success}`);
     }
 
+    private onVisibleCellsChange = (rowIndices: IRowIndices, columnIndices: IColumnIndices) => {
+        const { rowIndexStart, rowIndexEnd } = rowIndices;
+        const { columnIndexStart, columnIndexEnd } = columnIndices;
+        this.maybeLogCallback(`[onVisibleCellsChange] rowIndexStart=${rowIndexStart} rowIndexEnd=${rowIndexEnd}` +
```

Comment:
you can also use multiline strings in backticks, but i think it includes the newline